### PR TITLE
Add RTD translation to Brazilian Portuguese (PT-BR)

### PR DESCRIPTION
**Pull request by luana-martins**
Add RTD translation to Brazilian Portuguese (PT-BR).
I'm still working on the translation of detectors.po and bugDescriptions.po.

----

Make sure these boxes are checked before submitting your PR -- thank you!

- [X] Added an entry into `CHANGELOG.md` if you have changed SpotBugs code

### CODE REVIEW

File: docs/locale/pt_BR/LC_MESSAGES/faq.po [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+msgid ""
+"By default, SpotBugs assumes that any method invocation can throw an "
+"unchecked runtime exception. As a result, it may assume that an unchecked"
+" exception thrown out of the method could bypass a call to a ``close()`` "
+"method for a stream or database resource."
+msgstr ""
+"Por padrão, o SpotBugs assume que qualquer invocação de método pode "
+"lançar uma exceção de tempo de execução não verificada. Como resultado, "
+"pode assumir que uma exceção lançada para fora do método pode ignorar uma"
+" chamada a um método ``close ()``para um fluxo ou recurso de banco de "
```
Comment (KengoTODA) [inline docs/locale/pt_BR/LC_MESSAGES/faq.po:212]: reST parser needs a space around "``", or reports a warning like `/home/docs/checkouts/readthedocs.org/user_builds/spotbugs-pt-br/checkouts/1796/docs/faq.rst:61:<translated>:1: WARNING: Inline literal start-string without end-string.`

```suggestion
" chamada a um método ``close ()`` para um fluxo ou recurso de banco de "
```

File: docs/locale/pt_BR/LC_MESSAGES/use-plugin-on-sonarqube.po [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+"use another SpotBugs plugin, you need to build your own SonarQube plugin."
+" For detailed requirements on SonarQube plugins, see `the SonarQube "
+"official guidelines`_."
+msgstr ""
+"O `SpotBugs SonarQube Plugin  <https://github.com/spotbugs/sonar-"
+"findbugs>`_ usa os principais plugins do SpotBugs, como `fb-contrib <http://fb-"
+"contrib.sourceforge.net/>`_ and `Find Security Bugs "
+"<http://h3xstream.github.io/find-sec-bugs/>`_. No entanto, se quiser usar outro "
+"plugin, você precisa criar seu próprio plugin SonarQube. Para obter os requisitos "
+"detalhados dos plugins do SonarQube, consulte as diretrizes oficiais da SonarQube."
```
Comment (KengoTODA) [inline docs/locale/pt_BR/LC_MESSAGES/use-plugin-on-sonarqube.po:39]: [readthedocs reported](https://readthedocs.org/projects/spotbugs-pt-br/builds/15191127/) a warning as follows: `/home/docs/checkouts/readthedocs.org/user_builds/spotbugs-pt-br/checkouts/1796/docs/use-plugin-on-sonarqube.rst:4: WARNING: inconsistent references in translated message. original: ['`the SonarQube official guidelines`_'], translated: []`

We have a linked text `diretrizes oficiais do SonarQube` in at line 54, so we can wrap the same text in this line:

```suggestion
"detalhados dos plugins do SonarQube, consulte as `diretrizes oficiais do SonarQube`_."
```

If `da` is better than `do`, need to fix both of this line and line 54.

**Review by KengoTODA — CHANGES_REQUESTED** [review]
There are two build warnings, check my comments in this review.
I cannot review the translation itself, so I'm going to try finding reviewers if possible.

File: docs/locale/pt_BR/LC_MESSAGES/use-plugin-on-sonarqube.po (hunk 2 above)
Comment (luana-martins) [reply docs/locale/pt_BR/LC_MESSAGES/use-plugin-on-sonarqube.po:39]: Both ways are correct. For consistency with the rest of the page, we can use "do SonarQube" =)

**Review by KengoTODA — APPROVED** [review]
Confirmed that there is no build error nor warning reported by RTD:

https://readthedocs.org/projects/spotbugs-pt-br/builds/15205254/

**Review by KengoTODA — APPROVED** [review]

**Comment by KengoTODA** [comment]
@luana-martins Thanks for your contribution! We are going to release this document in the coming 4.5.0 release. 

Probably I will mention you when we got some issues reported about this translation, please help me if possible.

**Pull request merged**